com.unity.ml-agents/Runtime/Academy.cs

```diff
@@ -113,7 +122,7 @@ public bool IsCommunicatorOn
         // Signals to all the listeners that the academy is being destroyed
         internal event Action DestroyAction;
 
-        // Signals the Agent that a new step is about to start. 
+        // Signals the Agent that a new step is about to start.
```

Comment:
Should this (and others nearby) be in `<summary>` tags?

Reply:
internal variables/methods don't require `<summary>` tags to pass validation. Probably worth adding them, but I focused this PR on clearing package validation errors.

Reply:
I let these ones be because they were marked `internal`.